semantic/flatbuffers_test.go

```diff
@@ -1,57 +1,244 @@
-package semantic_test
+package semantic
 
 import (
 	"testing"
 
 	flatbuffers "github.com/google/flatbuffers/go"
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/flux/parser"
 	"github.com/influxdata/flux/semantic/internal/fbsemantic"
 )
 
-func TestFlatBuffers(t *testing.T) {
+// TODO(cwolff): There needs to be more testing here.  Once we can serialize an arbitrary semantic graph
+//   in Rust, we can get more complete coverage without having to create FlatBuffers by hand.
+
```

Comment:
Hopefully I'll have the Rust stuff done in a few days so we can see some more testing here.